DSharpPlus/Enums/MessageType.cs

```diff
@@ -68,6 +68,11 @@ public enum MessageType : int
         /// <summary>
         /// Message when a user follows a news channel.
         /// </summary>
-        ChannelFollowAdd = 12
+        ChannelFollowAdd = 12,
+
+        /// <summary>
+        /// Message when a user replies to another user
+        /// </summary>
+        Reply = 19,
```

Comment:
Style question: Do you guys prefer the trailing comma or prefer it to not be there.

Reply:
Prefer it not to be there.

Reply:
```suggestion
        Reply = 19
```